content/qcto/data-science-practitioner/km-01/km-01-kt05/_index.md

```diff
@@ -0,0 +1,47 @@
+---
+_db_id:
+content_type: topic
+ready: true
+title: Applications of Data Science
```

Comment:
The title is duplicated in both the title and the content header. It would be better to have only one.